app/ts/components/requirements.tsx

```diff
@@ -60,14 +60,36 @@ export const ParentRequirements = ( { checkBoxes, fuses } : { checkBoxes: Parent
 	</>
 }
 
+export const EnsRequirements = () => {
+	return <>
+		<p class = 'subdomain-header'> Ethereum Name Service </p>
+		<div class = 'grid-container'>
+			<div class = 'grid-item' style = 'justify-self: start'>
+				<div style = 'display: grid; grid-template-rows: auto auto;'>
+					<label class = 'form-control'>
+						<input type = 'checkbox' name = 'switch' class = 'check' checked = { false } disabled = { true }/>
+						<p class = 'paragraph checkbox-text requirement'>
+							Convince ENS DAO to fix the bug reported <a href = 'https://discuss.ens.domains/t/temp-check-executable-revoke-the-daos-ability-to-upgrade-the-name-wrapper/19920/8'>the-daos-ability-to-upgrade-the-name-wrapper</a> so they cannot rug you.
+						</p>
+					</label>
+				</div>
+			</div>
+		</div>
+	</>
+}
+
+
 export const Requirements = ({ checkBoxesArray } : { checkBoxesArray: OptionalSignal<CheckBoxes> }) => {
 	const allCheckBoxes = checkBoxesArray.deepValue
 	if (allCheckBoxes === undefined) return <></>
-	return <div class = 'grid-container-bordered'> { [...allCheckBoxes].reverse().map((check, index) => {
-		const fuses = getRequiredFusesWithoutApproval(allCheckBoxes.length - index - 1, allCheckBoxes.map((c) => c.domainInfo))
-		if (check.type === 'parent') return <ParentRequirements checkBoxes = { check } fuses = { fuses }/>
-		return <ChildRequirements checkBoxes = { check } fuses = { fuses }/>
-	}) } </div>
+	return <div class = 'grid-container-bordered'> <>
+		{ [...allCheckBoxes].reverse().map((check, index) => {
+			const fuses = getRequiredFusesWithoutApproval(allCheckBoxes.length - index - 1, allCheckBoxes.map((c) => c.domainInfo))
+			if (check.type === 'parent') return <ParentRequirements checkBoxes = { check } fuses = { fuses }/>
+			return <ChildRequirements checkBoxes = { check } fuses = { fuses }/>
+		}) }
+		<EnsRequirements/>
+	</></div>
```

Comment:
Unrelated to this PR, but you don't need an empty fragment here because you already have everything wrapped in a `<div>`.  `<></>` is only needed when you want to return multiple top-level elements, because components must return a single "fragment" (i.e., element) and so you can wrap them in a no-op element to get around this limitation.
```suggestion
	return <div class = 'grid-container-bordered'>
		{ [...allCheckBoxes].reverse().map((check, index) => {
			const fuses = getRequiredFusesWithoutApproval(allCheckBoxes.length - index - 1, allCheckBoxes.map((c) => c.domainInfo))
			if (check.type === 'parent') return <ParentRequirements checkBoxes = { check } fuses = { fuses }/>
			return <ChildRequirements checkBoxes = { check } fuses = { fuses }/>
		}) }
		<EnsRequirements/>
	</div>
```